### ISSUE TITLE
Add scale and expo functionality

### ISSUE THREAD

**Issue by DrewMcArthur** [hernejj/workspace-grid-cinnamon-applet#13]
Obviously scale and expo are overridden and don't work with this applet, so I'd just suggest adding a similar functionality.  Showing all open windows (or just windows in the current workspace), and showing all workspaces, similar to ubuntu's unity.

**Comment by hernejj** [comment]
Hmmm... possible. It might be easier to just add a method of triggering the originals though.

**Comment by DrewMcArthur** [comment]
Currently, you simply override the keyboard shortcuts for scale and expo, maybe it'd be better to add custom keyboard shortcuts? That way, the user could simply assign new shortcuts to scale and expo, so the functionality wouldn't be lost.  I have no idea about the difficulty, but if you're able to create a user-executable command to switch workspaces, then you could create that shortcut in the keyboard settings i'm sure.

**Comment by TomPastyrik** [comment]
Yes please! I love Cinnamon but I wouldn't be able to use it without your applet - strong habbits... Scale and expo would be much appreciated.

**Comment by riatzukiza** [comment]
yes, it is a serious muscle memory killer to not have the ability to switch keyboard shortcuts. At the moment it seems like you use the built-in cinnamon right/left keybindings, and create a special case for up and down, never adding a keybinding to the cinnamon registry, only listening on an event and doing somthing on that event. I use custom keybindings for switching (h/l), I'd like the ability to switch the up down to j/k. The vim in me is strong.

**Comment by riatzukiza** [comment]
so the program just straight up used the same bindings with out changing their name in the system? I changed the binding for "show expo" in my keyboard short cuts to ctrl+alt+k and I am now moving around. It would definitely be better to create custom shortcuts for this. I might get bored and fork this, I was poking around the code any way.

**Comment by meglington** [comment]
Thanks for the applet (hard to believe that grid navigation is not a built in option). Any update on this issue? I would also like the ability to assign Scale and Expo to a different key combo, so that we can have both up and down navigation, and the Scale and Expo features. Thanks
